src/NetTopologySuite.IO.GeoJSON4STJ/Converters/StjFeatureConverter.cs

```diff
@@ -107,14 +107,14 @@ public override IFeature Read(ref Utf8JsonReader reader, Type objectType, JsonSe
                                 break;
 
                             case JsonTokenType.Number:
-                                throw new NotSupportedException("Number value cannot be boxed as a decimal: " + reader.GetString());
+                                throw new ParseException(new NotSupportedException("Number value cannot be boxed as a decimal: " + reader.GetString()));
```

Comment:
the inner exceptions don't really need to be those types
```suggestion
                                throw new ParseException("Number value cannot be boxed as a decimal: " + reader.GetString());
```

Reply:
Actually I didn't want to change Stj classes in the first round at all. I'll revert these changes.